### DISCUSSION TITLE
How to download presidio images from microsoft private repository.

### DISCUSSION THREAD

**Discussion by vasanthv486** [microsoft/presidio#1313] [General]
Hi Team,

Please provide instructions on how to download presidio images from Microsoft repository.
mcr.microsoft.com

**Comment by omri374** [comment]
Hi, the Microsoft repository mcr.microsoft.com has the same images as Dockerhub, so if you don't have to access mcr.microsoft.com directly. If you want, you can just call `docker pull mcr.microsoft.com/presidio-analyzer:latest`
